src/ontology/Makefile

```diff
@@ -42,9 +42,9 @@ REPORT_FAIL_ON =            ERROR
 REPORT_LABEL =              -l true
 REPORT_PROFILE_OPTS =       --profile $(ROBOT_PROFILE)
 OBO_FORMAT_OPTIONS =        
-SPARQL_VALIDATION_CHECKS =  owldef-self-reference iri-range label-with-iri multiple-replaced_by term-editor-uri dce 
+SPARQL_VALIDATION_CHECKS =  owldef-self-reference iri-range label-with-iri multiple-replaced_by term-editor-uri dce obsolete obsolete-relation 
```

Comment:
Why `obsolete obsolete-relation`

Reply:
Sorry forgot to update repo. The first obsolete is for classes, but we don't have here.